eng/pipelines/templates/steps/ci.test.yml

```diff
@@ -136,27 +136,6 @@ steps:
   displayName: Validate Characters are ASCII
   condition: eq(variables['AZ_SDK_CODE_COV'], 1)
 
-# Validate all the files are formatted correctly according to the .clang-format file.
-- bash: |
-    # Run clang-format-9 recursively on each source and header file within the repo.
```

Comment:
Are we dropping clang-formatting at all, or some other CI leg is going to verify it?

This way or another, we should be using clang-format-18.

Reply:
Ah, sorry, that is C repo. I don't have as strong opinion for C repo, although 18 might be the most convenient, since it comes with current versions of VS, plus we would be using it in C++ repo.

Reply:
Bringing clang-format back is tracked here https://github.com/Azure/azure-sdk-for-c/issues/3099